pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -302,6 +337,9 @@ func (ca *cloudCIDRAllocator) updateCIDRAllocation(nodeName string) error {
 		return err
 	}
 
+	if northInterfaces != nil || additionalNodeNetworks != nil {
+		// TODO: perform IP capacity and annotation updates
```

Comment:
Let's at least put a github handle for the TODO? :)

Reply:
Done.